validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -464,6 +476,29 @@ def remove_temp_endpoint(self, endpoint):
         if endpoint in self._temp_endpoints:
             del self._temp_endpoints[endpoint]
 
+    def _check_temp_endpoints(self):
+        for endpoint in self._temp_endpoints:
+            endpoint_info = self._temp_endpoints[endpoint]
+            if (time.time() - endpoint_info.time) > \
+                    endpoint_info.retry_threshold:
+                LOGGER.debug("Endpoint has not completed authorization in %s "
+                             "seconds. Retry endpoint: %s ",
```

Comment:
There is an extra space at the end here.

Reply:
Maybe simplify this to "Endpoint has not completed authorization in %s sections: %s".

It is not generally common to use sentences in logging statements. (Though it's not a super strict rule.)